FFI/FFI.php

```diff
@@ -157,7 +157,7 @@ public static function typeof(CData $ptr): CType {}
          * dimensions specified by $dimensions.
          *
          * @param CType $type
-         * @param array|int[] $dimensions
+         * @param int[] $dimensions
```

Comment:
is it too early to use generics? `array<int>` etc?
I did run into https://youtrack.jetbrains.com/issue/WI-63759, so I guess yeah.

Reply:
`array<int>` has no advantages over `int[]` - and as you already menioned... its less supported in tooling